### ISSUE TITLE
Add flag to download RaspberryJuice plugin

### ISSUE THREAD

**Issue by itzg** [itzg/dockerfiles#88]
Provide a built-in flag to do the Bukkit plugin described in [the wiki](https://github.com/itzg/dockerfiles/wiki/Minecraft-Pi).

- [ ] Add plugin download flag
- [ ] Add a link to the wiki area in general in the README

**Comment by mcrowson** [comment]
Would love this

**Comment by itzg** [comment]
Yeah, I have been trying to think of a generic way to handle mod/plugin downloads; however, this one is likely worth specific handling since CurseForge/FTB modpacks do a good job of covering the majority cases.

**Comment by mcrowson** [comment]
Would a generic way just be to have an environment variable for plugin URLS to save to the plugins folder?

**Comment by itzg** [comment]
Ooh, good idea. I could probably make that work.

**Comment by mcrowson** [comment]
Any timeline on this or suggestions on how to implement right now?

I'm trying to get this all working with docker volumes and can't seem to get it. I've tried building my own image and getting the file after like so:

```Dockerfile
FROM itzg/minecraft-server
RUN curl -L -o /plugins/raspberryjuice-1.11.jar https://github.com/zhuowei/RaspberryJuice/raw/master/jars/raspberryjuice-1.11.jar 
```

but the file isn't in the plugins folder, I suspect it is getting overwritten somewhere else in the process. I also tried downloading it to `/data/plugins` but it also never shows up.
